namelist.wps

```diff
@@ -1,19 +1,21 @@
+! Add comments by prefacing with an "!" 
+
 &share
  wrf_core = 'ARW',
  max_dom = 2,
- start_date = '2006-08-16_12:00:00','2006-08-16_12:00:00',
- end_date   = '2006-08-16_18:00:00','2006-08-16_12:00:00',
+ start_date = '2019-09-04_12:00:00','2019-09-04_12:00:00',
+ end_date   = '2019-09-06_00:00:00','2019-09-06_00:00:00',
```

Comment:
Should we set `end_date` to the same as `start_date` for domain 2?

Reply:
@mgduda 
Probably. That's the way we've done it in the past, but let's get @weiwangncar opinion.

Reply:
Michael and Kelly,
I think we can set end_date to the same as start_date for D02. This is because we only needs met_em file at the initial time for D02.

Reply:
I am ok with that. Thanks.